docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device/performance-hint-and-threads-scheduling.rst

```diff
@@ -0,0 +1,163 @@
+
+Throughput Hint
+######################################
+
+In this scenario, thread scheduling first evaluates the memory pressure of the model being inferred on the current platform, and determines the number of threads per stream, as shown below.
+
++-----------------+-----------------------+
+| Memory Pressure | Threads per stream    |
++=================+=======================+
+| low             | 1 P-core or 2 E-cores |
++-----------------+-----------------------+
+| medium          | 2                     |
++-----------------+-----------------------+
+| high            | 3 or 4 or 5           |
++-----------------+-----------------------+
+
+Then the value of ``ov::num_streams`` is calculated as ``ov::inference_num_threads`` divided by the number of threads per stream. The default settings of low-level performance properties on Windows and Linux are as follows:
+
++--------------------------------------+-------------------------------+-------------------------------+
+| Property                             | Windows                       | Linux                         |
++======================================+===============================+===============================+
+| ``ov::num_streams``                  | Calculate as above            | Calculate as above            |
++--------------------------------------+-------------------------------+-------------------------------+
+| ``ov::inference_num_threads``        | Number of P-cores and E-cores | Number of P-cores and E-cores |
++--------------------------------------+-------------------------------+-------------------------------+
+| ``ov::hint::scheduling_core_type``   | P-cores and E-cores           | P-cores and E-cores           |
++--------------------------------------+-------------------------------+-------------------------------+
+| ``ov::hint::enable_hyper_threading`` | Yes / No                      | Yes / No                      |
++--------------------------------------+-------------------------------+-------------------------------+
+| ``ov::hint::enable_cpu_pinning``     | No                            | Yes                           |
++--------------------------------------+-------------------------------+-------------------------------+
+
+.. note::
+
+    - By default, each stream is created on P-core only or E-core only on single numa node even all P-cores and E-cores are used in this scenario.
+    - By default, ``ov::hint::enable_hyper_threading`` is ``Yes`` on single socket platform to run more streams in parallel. And it is ``No`` on dual sockets platform because dual sockets platform has more CPU cores and streams already.
```

Comment:
To be honest such explanation doesn't make any sense. Maybe better to delete at all.